### PR TITLE
README, .env, config

### PR DESCRIPTION
**Pull request by HuckMatt**
Updated readme to standard format

removed .env and provide details based in .env.example

update next.config.js to get auth details from .env

### CODE REVIEW

**Comment by HuckMatt** [comment]
@eric-giannini please review.

File: README.md [hunk 1]
```diff
@@ -1,71 +1,85 @@
...
+2. To run the app use `yarn dev`.
+3. Navigate to `localhost:3000` to view the app.
 
-### Index page
 
-On the first page of the app we showcase how you can call to your phone number (or meeting number) by providing the phone and multiple parameters. You will get conversational insights, transcriptions, messages, which `Symbl.ai` gets from this call. Architecture looks similar to this:
+## Integration 
 
-![](https://docs.symbl.ai/images/tutorial_phone_integration-f54ba415.png)
+### Project Structure
```
Comment (eric-giannini) [inline README.md:47]: May you please add to TOC?

File: README.md [hunk 2]
```diff
@@ -1,71 +1,85 @@
...
 
-- `pages/api` here reside NextJS API routes. Here we have `call` and `subscribeToPhoneEvents` routes that will be translated to `/api/call` and `/api/subscribeToPhoneEvents` paths in the browser. In this routes we showcase how you can use Symbl Node SDK to get realtime data from API. You can read more about NodeSDK [here](https://docs.symbl.ai/#symbl-sdk-node-js) or read a how to guide [here](https://docs.symbl.ai/#get-live-transcription-phone-call-node-js-telephony)
+1. Run `yarn install` or `npm install`.
+2. To run the app use `yarn dev`.
+3. Navigate to `localhost:3000` to view the app.
 
-### Index page
 
-On the first page of the app we showcase how you can call to your phone number (or meeting number) by providing the phone and multiple parameters. You will get conversational insights, transcriptions, messages, which `Symbl.ai` gets from this call. Architecture looks similar to this:
+## Integration
```
Comment (eric-giannini) [inline README.md:45]: May you please add to TOC?

File: README.md [hunk 3]
```diff
@@ -1,71 +1,85 @@
...
+ * [Setup](#setup)
+ * [Integration](#integration) 
+ * [Conclusion](#conclusion)
+ * [Community](#community)
+ 
+## Setup 
+The first step to getting setup is to [sign up][signup] with Symbl.ai. 
 
-## About this app.
+### Authorization
```
Comment (eric-giannini) [inline README.md:31]: May you please add to TOC?

**Comment by eric-giannini** [comment]
LGTM, @HuckMatt Please update so we can merge.

**Pull request merged**